Add note to Jobs and ISPs

kitten opened the pull request.

kitten committed:
Add note to Jobs and ISPs

lorey commented:
You wrote:

> they allow you to cancel the contract early (though it's not they're not obligated to!

In fact, ISPs in Germany are obligated to cancel your account if they cannot provide their service at your new location:

> Grundsätzlich gilt für das Thema DSL Kündigung beim DSL Umzug: Steht ein Wechsel des Wohnorts an, zieht der DSL Anschluss mit um. Seitdem das Telekommunikationsgesetz aus dem Mai 2012 in Kraft getreten ist, gilt jedoch: Kann der Provider am neuen Wohnort kein entsprechendes Komplettpaket schalten, greift das DSL Sonderkündigungsrecht. Sollte also der Check der DSL Verfügbarkeit am neuen Wohnsitz negativ ausfallen, kann der bisherige Vertrag ohne Sonderzahlungen gekündigt werden.
> Aber auch wenn der DSL Anbieter in der Lage ist, am neuen Wohnort einen Internetanschluss bereitzustellen, kann eine Sonderkündigung gerechtfertigt sein. Dies ist der Fall, wenn der neue Anschluss nicht die gleiche Leistung erbringt, wie der bisher genutzte, z.B. wenn statt 16 Mbit/s nur noch 6 Mbit/ an Bandbreite zur Verfügung stehen.

See [this article in German](https://www.dslweb.de/dsl-kuendigung-vertrag-bei-umzug-kuendigen.php)

kitten commented:
@lorey I didn't know that they were actually obligated in that case. I'll edit this then :)

Commit:
Fix some grammatical mistakes and typos

Commit:
Add notes for ISP contract term and Sonderkündigungsrecht

Commit:
Correct trivia on ISP flat limits

Most flats are actually not limited and are in fact a proper unlimited
flat.
Having limited packages / contracts is not unheard of, but not the rule.

Commit:
Add more explicit note on file-sharing

kitten commented:
- Added some notes on the DSL Sonderkündigungsrecht
- Corrected the trivia on the ISP's data limits
- Added a separate trivia on file-sharing (It's monitored quite heavily in Germany and I know lots of people who received letters from lawyers)

lorey commented:
Good idea to point out the danger of getting sued for file-sharing! This topic comes up all the time with our programmers. Thanks a lot.

azproduction commented:
Thank you for updates!
I also have Telecolumbus and if I completely disable the integrated Wifi module it works 24/7 like a clock.

kitten commented:
@azproduction I did run it just as a modem and connected my own router. That reminds me... In Berlin the 2.4 GHz bands are really crowded. It might be worth adding some notes on that?

lorey commented:
Thanks for the great additions! 
I think crowded bandwidths are not specific to Germany or relocation and techies should figure that out without any HowTo.
